### ISSUE TITLE
Some units in defaultUnits incorrect

### ISSUE THREAD

**Issue by ambaker1** [OpenSees/OpenSees#412]
Hello, I have been doing work on my own unit procedures for OpenSees and decided to compare against the "defaultUnits" command in OpenSees. I know it is not used widely in the Tcl scripting community, but I saw a number of uses in Python scripts. So, I thought I would bring this to attention.

1. The conversion for mm is incorrect. It uses in/25.44, should be in/25.4
https://github.com/OpenSees/OpenSees/blob/69a04c0d1d8f09266f25d822a260d79538cd8d4b/SRC/interpreter/OpenSeesCommands.cpp#L2872
This will carry over to units mm2 and mm4 as well.

2. The value of lbs in kgf is incorrect. It multiplies when it should divide gravity.
https://github.com/OpenSees/OpenSees/blob/69a04c0d1d8f09266f25d822a260d79538cd8d4b/SRC/interpreter/OpenSeesCommands.cpp#L2808
Derivation:

```
lbf = 4.4482216152605*kg*m/s^2
kgf = kg*g
g = 9.80665*m/s^2
kg = kgf/g
lbf = (4.4482216152605/9.80665)*kgf
```

3. The value of lbs in tonf is incorrect. This problem is the same as with kgf, since tonf = 1000*kgf
https://github.com/OpenSees/OpenSees/blob/69a04c0d1d8f09266f25d822a260d79538cd8d4b/SRC/interpreter/OpenSeesCommands.cpp#L2811

4. The handling of temperature units is not useful at all. Variables F & C are assigned values, even though they have different scales. What is actually returned to the user is 1 for the default unit and the converted value of 1 in the other temperature scale. Not very useful.
Example script:
```
puts "--Default temperature unit Celsius--"
defaultUnits -Force kip -Length in -Time sec -Temp C
puts " Fahrenheit:\t[format %f $F]"
puts " Celsius:\t[format %f $C]"

puts "--Default temperature unit Fahrenheit--"
defaultUnits -Force kip -Length in -Time sec -Temp F
puts " Fahrenheit:\t[format %f $F]"
puts " Celsius:\t[format %f $C]"
```
This returns:
--Default temperature unit Celsius--
 Fahrenheit:    33.800000
 Celsius:       1.000000
--Default temperature unit Fahrenheit--
 Fahrenheit:    1.000000
 Celsius:       -17.222222

While the other unit variables work as the conversion, I cannot take my Fahrenheit temperature, multiply it by 33.8, and get Celsius. Therefore, I suggest to either use only Rankine and Kelvin, which have the same zero point, or ditch the temperature unit entirely.

**Comment by jaabell** [comment]
I had no idea about these commands! I don't see any problem with your suggestion.

**Comment by mhscott** [comment]
Fixed #505
